src/server/server.ts

```diff
@@ -48,6 +46,88 @@ export class GraphQLTransportWSServer {
   private wsServer: WebSocket.Server;
```

Comment:
let's rename onSubscribe and onUnsubscribe to onRequest and onRequestComplete
and add a wrapper named onSubscribe/Unsubscribe to emit deprecation warnings.